### PR TITLE
Task 4 - Log in API for user

### PR DESCRIPTION
**Pull request by Vaishnavi-Mhetre**
Created route for log in api and added function in user controller.

### CODE REVIEW

File: controllers/userController.js [hunk 1]
```diff
@@ -57,6 +57,42 @@ const signUp = asyncHandler(async (req,res) => {
...
+        }) 
+     }
+
+    const user = await User.findOne({email})
+    if(!user){
+        res.status(400).json({
+            message : "User not found"
+        }) 
+    }
+    console.log("password", password)
```
Comment (Sammed7) [inline controllers/userController.js]: Please remove consoles.

File: controllers/userController.js [hunk 2]
```diff
@@ -57,6 +57,42 @@ const signUp = asyncHandler(async (req,res) => {
...
+    const DBpassword = await user.password
+    console.log("dbpass", DBpassword)
+    console.log(await bcrypt.compare(password, DBpassword))
```
Comment (Sammed7) [inline controllers/userController.js]: Please remove consoles from here also.

Comment (Vaishnavi-Mhetre) [reply controllers/userController.js]: removed.

File: controllers/userController.js (hunk 1 above)
Comment (Vaishnavi-Mhetre) [reply controllers/userController.js]: removed.

**Review by Sammed7 — APPROVED** [review]

**Pull request merged**